src/schema/slack/types/custom/form_input.ts

```diff
@@ -18,6 +19,7 @@ const FormInput = DefineType({
       },
     },
   },
+  required: [],
```

Comment:
I think both of these are required, but we may need to double check

Reply:
How/where can I double check?

Reply:
I checked by testing an app that uses `OpenForm` (which uses `FormInput`) and commented out both `elements` and `required`. Turns out `elements` is required, but `required` is optional! Updated accordingly